Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM ruby:3.0.0
 
-ENV KUBECTL_VERSION=v1.18.2
+ENV KUBECTL_VERSION=v1.20.2
 ENV EJSON_VERSION=1.2.1
```

Comment:
There is `1.2.2` available

Reply:
For some reason it's not available on rubygems, so I updated Dockerfile to fetch it from their github releases (which seems like a [suggested installation way](https://github.com/Shopify/ejson#installation) nowadays).

Reply:
🤯